ChangeLog.d/tls13-fix-finished-fetch.txt

```diff
@@ -1,5 +1,3 @@
 Bugfix
-   * Fix handshake failure when the peer Finished message has not been received
-     yet when we first try to fetch it. The fetching is moved before the
-     preprocessing computations to avoid doing them multiple times, which was
-     causing the handshake to fail.
+   * Fix a TLS 1.3 handshake failure when the peer Finished message has not
+     been received yet when we first try to fetch it.
```

Comment:
I understand that we don't have to give more detail, but is it harmful if the work is already done?

Reply:
Having irrelevant information hides the relevant information, yes. So I'm not reverting this.